index.html

```diff
@@ -662,17 +570,7 @@ <h1>Terminology</h1>
             </dt>
             <dd>An entity that exposes a network interface
                 consistent with a WoT Thing Description. WoT Server is
-                also used to refer to a Servient in server role only.</dd>
-            <dt>
-                <dfn>WoT Servient</dfn>
-            </dt>
```

Comment:
It would be pragmatic to have both terms WoT Servient and Servient defined in the document.